Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs

```diff
@@ -180,8 +180,27 @@ public MixedRealityControllerMappingProfile ControllersProfile
         /// </summary>
         public bool EnableBoundarySystem
         {
-            get { return enableBoundarySystem; }
-            private set { enableBoundarySystem = value; }
+            get
+            {
+                return boundarySystemType != null &&
```

Comment:
Redundant first check?

Reply:
Unity will let you select NONE as the SystemType, so we have to check if the user has selected something.
This is the same code as the InputSystem config uses.

Reply:
Thoughts @StephenHodgson ? As this was cloned from the Input System code

Reply:
Sorry, I meant the null conditional operator below it, `boundarySystemType` is being checked null even though the operator below does that check.

Reply:
It looks like @Jarodshow is on to something:

Console.WriteLine("TestObject != null              : {0}", to != null);
Console.WriteLine("TestObject?.InnerObject != null : {0}", to?.InnerObject != null);

Results in:

TestObject != null              : False
TestObject?.InnerObject != null : False